community/contribute/scholarships.md

```diff
@@ -0,0 +1,13 @@
+# Scholarships
+
+The [PEP-31 Scholarship Fund](https://forum.pokt.network/t/pep-31-scholarship-fund/) was [approved](https://gov.pokt.network/#/proposal/0x2920a4057b4aa6af033654bfcfdd77f2b98c81f100d8e55b97a30101e563db46) by the DAO in 06/2022 to allocate an accumulating monthly allowance of `10,000/POKT` per month to contributors, students, learners, speakers, educators and others or in the Pocket community.
```

Comment:
* Suggest "June 2022", if we need a date at all (I'm not sure we do).
* Suggest `10,000 POKT`.
* Stray word: "...educators, and others in the Pocket community."